docs/index.rst

```diff
@@ -3,6 +3,8 @@
 .. include:: ../README.rst
   :start-after: inclusion-marker-do-not-remove
 
+.. Note: variables.rst is autogenerated by write_variables.py
```

Comment:
just a note for the future, when we add other parametrizations it might be best to have multiple tabs under List of Variables, one for each parametrization. Either that or have another column where it lists which objects are able to calculate each variable, otherwise the list page will get really long